File: viasoft.qualidade.rnc-frontend/apps/rnc/src/api-clients/Nao-Conformidades/Operacao-Retrabalho/model/maquina-input.ts
import { MaterialInput } from './material-input';

export class MaquinaInput {
  public id: string;
  public idRecurso: string;
  public descricao: string;
  public horas: number;
  public minutos: number;
  public detalhamento: string;
  public materiais: Array<MaterialInput>;
}

File: viasoft.qualidade.rnc.gateway/Viasoft.Qualidade.RNC.Gateway.Host/NaoConformidades/RetrabalhoNaoConformidades/OperacoesRetrabalhoNaoConformidade/Dtos/MaquinaInput.cs
using System;
using System.Collections.Generic;

namespace Viasoft.Qualidade.RNC.Gateway.Host.NaoConformidades.RetrabalhoNaoConformidades.OperacoesRetrabalhoNaoConformidade.Dtos;

public class MaquinaInput
{
    public Guid Id { get; set; }
    public string Detalhamento { get; set; }
    public int Horas { get; set; }
    public int Minutos { get; set; }
    public Guid IdRecurso { get; set; }
    public List<MaterialInput> Materiais { get; set; } = new();
}